agent_central/services/hq_service.py

```
@@ -367,21 +536,32 @@ def _upskill_role(self, role_name: str, knowledge_content: str):
         else:
-            updated_content = current_content + "\n\n## 7. Learned Protocols\n" + synthesized_block
-            
+            updated_content = (
+                current_content + "\n\n## 7. Learned Protocols\n" + synthesized_block
+            )
+
         role_file.write_text(updated_content, encoding="utf-8")
+
+    def _load_config(self, config_file: Path) -> AgencyConfig:
+        try:
+            with open(config_file, "r", encoding="utf-8") as f:
```

Ruff (0.14.14)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)